lxd/db/transaction.go

```diff
@@ -53,3 +53,12 @@ func (c *ClusterTx) stmt(code int) *sql.Stmt {
 	}
 	return c.tx.Stmt(stmt)
 }
+
+// prepare prepares a new statement from a SQL string.
+func (c *ClusterTx) prepare(sql string) (*sql.Stmt, error) {
+	stmt, err := c.tx.Prepare(sql)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to prepare statement with error: %v", err)
+	}
+	return stmt, nil
```

Comment:
@masnax just a nit, but something I've noticed you do a few times now, can you add a newline after each block, so here it would be:

```go
        stmt, err := c.tx.Prepare(sql)
	if err != nil {
		return nil, fmt.Errorf("Unable to prepare statement with error: %v", err)
	}

	return stmt, nil
```

Cheers